covid-vaccine-faq-bot/assets/index.html

```diff
@@ -0,0 +1,91 @@
+                Consumers send their questions through SMS to a pre-configured phone number, and are responded by FAQ chatbot which leverages official sources(CDC, WHO) for answers and is powered by a Twilio Studio Flow.</p>
+        </header>
+        <main>
+            <section>
+                <div class="loader"></div>
+                <div id="deploy-flow" style="display: none;" class="deploy-studio-flow">
+                  <h3>Finish configuring your app</h3>
+                  <p>Your app is deployed, but you need to finish configuring it to get it working. Click the button below to automatically run the following deployment steps:</p>
+                  <ul>
+                    <li>Deploy a Twilio Studio Flow to power your Vaccine FAQ SMS Bot</li>
+                    <li>Link your Twilio phone number (<strong class="phone-number"></strong>) with Studio Flow</li>
+                  </ul>
+                  <a class="button" onclick="setup(event)">Deploy Studio Flow</a>
+                  <div class="loader button-loader" style="display: none;"></div>
+                </div>
+                <div id="flow-deployed" style="display: none;" class="deploy-studio-flow">
+                  <h3><span class="checkmark">&#10003;</span> Your Studio Flow is deployed</h3>
+                  <p>You can move on to test your vaccine faq bot below or click the button to make changes to your Studio Flow.<br/>
+                  <a id="open-studio" class="button button-outline" href="" target="_blank">Edit Studio Flow</a>
+                </div>
+              </section>
+              <div class="post-deploy-studio" style="display:none;">
+                <section>
+                  <h3>Test your Vaccine FAQ Bot</h1>
+                  <p>You can try out your Vaccine FAQ Bot by sending a text message to <strong class="phone-number"></strong>. If your account is in trial mode, you will need to send the SMS from <a href="https://www.twilio.com/docs/usage/tutorials/how-to-use-your-free-trial-account#verify-your-personal-phone-number" target="_blank">verified phone numbers</a>.</p>
+                </section>
+                <section>
+                  <h3>Next Steps</h3>
+                  <p>Getting the world vaccinated is no small feat. We're here to help. <strong><a href="https://ahoy.twilio.com/vaccine-distribution-1" target="_blank">Get in touch</a></strong> if you would like support with any of the following on the path to a production deployment:</p>
+                  <ul>
+                    <li>Accessing information in local languages depending on the country</li>
+                    <li>Adding other channels for for getting answers like Voice, WhatsApp, and Web Chat</li>
+                    <li>Accessing implementation support from Twilio's trusted partners</li>
+                  </ul>
+                </section>
+                <section>
+                  <h3>Troubleshooting</h3>
```

Comment:
One thing I'd recommend stealing from the google-sheets template and adding here is a way to verify that your Google Authentication is set up properly. 

See [here](https://github.com/twilio-labs/function-templates/blob/main/google-sheets/assets/index.html#L112-L119) for details. Essentially the client calls out to a function that verifies everything looks OK.

Reply:
this has been created and merged